js/main.js

```diff
@@ -263,10 +361,18 @@ class App extends React.Component {
     }
   };
 
+  setupPolling = () => {
+    this._socket = new Poller(this);
+  };
+
   connect = () => {
     if (this._socket) {
       return;
     }
+    if (USE_POLLING == 'True') {
```

Comment:
Is it an easy fix to have this be set to the js native value of `true`? This seems a bit magic string-y.

Reply:
It just amounts to changing the location where I do the magic. I'll put it in the place where USE_POLLING is set